packages/cli/stores/config.ts

```diff
@@ -89,11 +91,25 @@ class ConfigStore {
       return;
     }
 
+    // Load the client version from the module's package.json metadata
     try {
-      const packageJSONPath = await findUp("package.json");
+      const moduleMetadata = await readFile(
+        join(moduleRoot, "package.json"),
+        "utf-8",
+      );
+      const moduleMetadataParsed = parseJSON(moduleMetadata) as unknown as {
+        version: string;
+      };
+      this.clientVersion = moduleMetadataParsed.version;
```

Comment:
this is probably fine, but as a point of reference this is what the SDKs do: https://github.com/bucketco/bucket-javascript-sdk/blob/main/packages/browser-sdk/src/config.ts#L1

Reply:
We can't unfortunately do that without getting the "importing json is experimental warning" because the package uses type "module"